content/agent/_index.md

```diff
@@ -3,103 +3,137 @@ title: Agent
 
-[The source code for the Datadog Agent](https://github.com/DataDog/dd-agent).
+{{< partial name="platforms/platforms.html" >}}
 
-For information on running the Agent through a proxy, see [the dedicated documentation Datadog Agent and proxy ](/agent/proxy); for which IP ranges to allow, see [the complete list of IP's and ports](https://github.com/DataDog/dd-agent/wiki/Network-Traffic-and-Proxy-Configuration#open-ports).
+## Start/Stop/Restart the agent 
+### Start the agent
 
-The Agent has three main parts: the collector, DogStatsD, and the forwarder:
+|Platform|Agent v5 |Agent v6|
+|:--------|:-----|:--------|
+|Linux|`sudo service datadog-agent start`|`sudo service datadog-agent start`|
+|MacOS x|`/usr/local/bin/datadog-agent start`|`launchctl start com.datadoghq.agent` or systray app |
+|SmartOS|`svcadm enable datadog`||
+|Source|`sudo ~/.datadog-agent/bin/agent start`|`sudo service datadog-agent start`|
+|Windows|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|
 
-* **The collector**: runs checks on the current machine for whatever [integrations](/integrations) you have and it captures system metrics such as memory and CPU.
+### Stop the agent
 
-* **DogStatsD**: It is a statsd backend server you can send [custom metrics](/getting_started/custom_metrics/) to from an application.
+|Platform|Agent v5 |Agent v6|
+|:--------|:-----|:--------|
+|Linux|`sudo service datadog-agent stop`|`sudo service datadog-agent stop`|
+|MacOS x|`/usr/local/bin/datadog-agent stop` |`launchctl stop com.datadoghq.agent` or systray app  |
+|SmartOS|`svcadm disable datadog`||
+|Source|`sudo ~/.datadog-agent/bin/agent stop`|`sudo service datadog-agent stop`|
+|Windows|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|
+### Restart the agent 
 
-* **The forwarder**: retrieves data from both DogStatsD and the collector and then queues it up to be sent to Datadog.
+|Platform|Agent v5 |Agent v6|
+|:--------|:-----|:--------|
+|Linux|`sudo service datadog-agent restart`|`sudo service datadog-agent restart`|
+|MacOS x|`/usr/local/bin/datadog-agent restart `|_run `stop` then `start`_ or systray app|
+|SmartOS|`svcadm restart datadog`|`sudo service datadog-agent restart`|
+|Source|`sudo ~/.datadog-agent/bin/agent restart`||
+|Windows|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows)|
 
-This is all controlled by one supervisor process. We keep this separate so you don't have to have the overhead of each application if you don't want to run all parts, although we generally recommend you do.
+## Agent Status and information
 
-<div class="alert alert-info">
-This documentation covers Agent versions 5.0.0 and above.
-</div>
+### Agent Status
```

Comment:
I'd change to "Service status"

Reply:
let's not confuse our users. even if technically you are right.

Reply:
Not sure I understand the confusion, but I dont have a strong opinion here.

Reply:
`Service status` makes more sense to me as well.